### PR TITLE
[workspace]fix the workspace selector jump that occurs when hovering and clicking.

### PR DESCRIPTION
**Pull request by Qxisylolo**
### Description

This pr fixes the workspace selector jump that occurs when hovering and clicking.

## Screenshot
Before:

https://github.com/user-attachments/assets/ac72b982-87f1-4fd6-885b-d8c242bb07bd

After:

https://github.com/user-attachments/assets/e3aecc24-9d0b-4934-a8d4-9846f0c83952

## Changelog
- fix: the workspace selector jump that occurs when hovering and clicking.

### Check List

- [ ] All tests pass
  - [ ] `yarn test:jest`
  - [ ] `yarn test:jest_integration`
- [ ] New functionality includes testing.
- [ ] New functionality has been documented.
- [ ] Update [CHANGELOG.md](./../CHANGELOG.md)
- [ ] Commits are signed per the DCO using --signoff

### CODE REVIEW

File: src/plugins/workspace/public/components/workspace_selector/workspace_selector.tsx [hunk 1]
```diff
@@ -66,6 +66,7 @@ export const WorkspaceSelector = ({ coreStart, registeredUseCases$ }: Props) =>
   };
   const button = currentWorkspace ? (
     <EuiPanel
+      className="popoverButton"
```
Comment (SuZhou-Joe) [inline src/plugins/workspace/public/components/workspace_selector/workspace_selector.tsx:69]:
```suggestion
      className="workspaceSelectorPopoverButton"
```
Adding some prefix makes me feel safer, can we?

Comment (Qxisylolo) [reply src/plugins/workspace/public/components/workspace_selector/workspace_selector.tsx]: Thanks for the advice, always updated.

**Review by SuZhou-Joe — DISMISSED** [review]

**Review by SuZhou-Joe — APPROVED** [review]

**Review by Hailong-am — APPROVED** [review]
need confirm whether we need the shadow when hover

**Comment by ruanyl** [comment]
The animation is the default behavior of a clickable OuiPanel, instead of override the styles from component library, maybe we shouldn't use `OuiPanel` for this component

**Review by ruanyl — APPROVED** [review]

**Pull request merged**